scripts/zones/Abyssea-Vunkerl/npcs/Cruor_Prospector.lua

```diff
@@ -0,0 +1,116 @@
+-----------------------------------
+-- Area: Abyssea - Vunkerl
+--  NPC: Cruor Prospector
+-- Type: Cruor NPC
+-- @pos -318.000 -40.949 685.500 217
+-----------------------------------
+package.loaded["scripts/zones/Abyssea-Vunkerl/TextIDs"] = nil;
+-----------------------------------
+require("scripts/zones/Abyssea-Vunkerl/TextIDs");
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/keyitems");
+require("scripts/globals/abyssea");
+
+-----------------------------------
+-- onTrade Action
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end;
+
+-----------------------------------
+-- onTrigger Action
+-----------------------------------
+
+function onTrigger(player,npc)
+    local Cruor = player:getCurrency("cruor");
+    local Demilune = getDemiluneAbyssite(player);
+    player:startEvent(2002, Cruor, Demilune);
+end;
+
+-----------------------------------
+-- onEventUpdate
+-----------------------------------
+
+function onEventUpdate(player,csid,option)
+    -- printf("CSID: %u",csid);
+    -- printf("RESULT: %u",option);
+end;
+
+-----------------------------------
+-- onEventFinish
+-----------------------------------
+
+function onEventFinish(player,csid,option)
+    -- printf("CSID: %u",csid);
+    -- printf("RESULT: %u",option);
+
+    if (option == 65539) then -- Map of Abyssea - Vunkerl
+        if (player:hasKeyItem(MAP_OF_ABYSSEA_VUNKERL) == false) then
```

Comment:
@xdemolish  The keyItems change with each zones's prospector, and the status effects used addStatusEffextEx dummy effects and have multiple effects of differing power per option so KI's and statusEffects are gonna be a little more involved to do what you wanted, I can't do that anytime soon so gonna have to take it or leave it bud. 

I still got rid of a crapton of it, so better than it was? >.>